hackathon_site/hardware/test_api.py

```diff
@@ -493,8 +500,28 @@ def test_user_has_no_view_permissions(self):
         response = self.client.get(self.view)
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
-    def test_user_has_view_permissions(self):
-        self._login(self.permissions)
+    def test_user_has_view_permissions_no_profile(self):
+        self._login(self.view_permissions)
+        response = self.client.get(self.view)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
```

Comment:
If they have view permissions, they should be given access. Tech team won't have a profile (though technically they could), but should be able to view orders.

Conversely, regular participants won't have view permissions, but they'll get to see their own orders because they have a profile.